com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DoFGather.compute

```diff
@@ -34,7 +34,6 @@ RW_TEXTURE2D_X(CTYPE, _OutputTexture);
 // A set of Defines to fine-tune the algorithm
 #define ADAPTIVE_SAMPLING
 #define STRATIFY
-#define FORCE_POINT_SAMPLING
```

Comment:
If I remember correctly, trilinear sampling can introduce some "leaking" between in-focus and out-of-focus regions, that's why I was forcing point sampling. Ideally this choice should be per tile (use trilinear only in tiles without in-focus pixels), but since we don't do that now, I think it's more safe to use point sampling.